pkg/util/globalerror/user.go

```diff
@@ -74,8 +74,9 @@ const (
 	StoreConsistencyCheckFailed ID = "store-consistency-check-failed"
 	BucketIndexTooOld           ID = "bucket-index-too-old"
 
-	DistributorMaxWriteMessageSize         ID = "distributor-max-write-message-size"
-	DistributorMaxWriteRequestDataItemSize ID = "distributor-max-write-request-data-item-size"
+	DistributorMaxWriteMessageSize                 ID = "distributor-max-write-message-size"
+	DistributorMaxOtelDecompressedWriteMessageSize ID = "distributor-max-otel-decompressed-write-message-size"
```

Comment:
```suggestion
	DistributorMaxOtlpRequestSize ID = "distributor-max-otlp-request-size"
```

Reply:
renaming is done in commit https://github.com/grafana/mimir/commit/1ba1bcb4854d016dfada97077fc1696ea23c84e3